superset-frontend/packages/superset-ui-core/src/chart/models/ChartProps.ts

```diff
@@ -105,6 +105,8 @@ export interface ChartPropsConfig {
   inputRef?: RefObject<any>;
   /** Theme object */
   theme: SupersetTheme;
+  /** Set chart data selectable by user */
+  dataSelectionMode?: string;
```

Comment:
It seems more appropriate to pass a more shareable value like canExportData rather than sending specific data selection values, and then set it according to each component's needs.

```suggestion
  canExportData: boolean;
```